Update `Projects` (#5708)
ppy/osu-wiki: wiki/Projects/en.md

## Official

> - ~~[Graveyarded Beatmap Archiving](https://osu.ppy.sh/community/forums/topics/49310)—A forum thread to request deleted graveyarded beatmaps.~~ Closed as of May 3, 2019. If you have a **very urgent** request you can send an email to peppy.

Reviewer: do not cross out

- [Live Event Feed](https://osu.ppy.sh/community/forums/topics/9015)—A live feed of current player events.
  - [Website](https://osu.ppy.sh/p/events)
- [osu! (OS X)](https://osu.ppy.sh/community/forums/topics/56460)—osu! on OS X (results will vary).
  - [GitHub](https://github.com/ppy/osu-wine) | [Website](http://osuosx.tumblr.com)

> - ~~[osu! Community Localisation Project](https://osu.ppy.sh/community/forums/topics/104342)—The osu! localisation project, localise your language into osu!.~~ Inactive, please use the [osu-wiki Github](https://github.com/ppy/osu-wiki/) or participate in the [osu-web Crowdin translations](https://crowdin.com/project/osu-web) instead.

Reviewer: same

### Beatmap Mirrors

- ~~BloodCat—A well known beatmap mirror hosted in Korea.~~ Shutdown as of January 2, 2021

>   - ~~[Website](https://bloodcat.com/)~~

Reviewer: same

*See also: [Internet Relay Chat](/wiki/Internet_Relay_Chat)*

- [chat4osu!](https://osu.ppy.sh/community/forums/topics/879262)—A chat client for referees and casual chatters.
  - [Gitlab](https://gitlab.com/hallowatcher/chat4osu)
- ~~[irc4osu!](https://osu.ppy.sh/community/forums/topics/430741)—A cross-platform IRC client for osu!.~~ Abandoned as of September 30, 2019

>   - ~~[GitHub](https://github.com/arogan-group/irc4osu)~~

Reviewer: same

- [osu! Stats Watcher](https://osu.ppy.sh/community/forums/topics/269483)—Write player stats into a file, which can then be displayed using Open Broadcaster Software.
  - [GitHub](https://github.com/albinohat/osu_stats_watcher)
- osu!chan—Hosts a multitude of alternative ranking leaderboards.
  - [Website](https://osuchan.syrin.me/)
- [osu!Skills](https://osu.ppy.sh/community/forums/topics/367283)—Expands pp into include other metrics such as stamina, reaction, and memory.
  - [Website](http://osuskills.com/)
- [osu!Stats](https://osu.ppy.sh/community/forums/topics/51911)—Search scores and player statistics.
  - [Website](https://osustats.ppy.sh/)
- [osu!track](https://osu.ppy.sh/community/forums/topics/212151)—A tool that allows players to track their progress in-game and see visually see their improvement over time.
  - [GitHub](https://github.com/Ameobea/osutrack_irc_v2/) | [Website](https://ameobea.me/osutrack)
- [osu!track (IRC)](https://osu.ppy.sh/community/forums/topics/233167)—The in-game IRC bot for osu!track.
- [osu!widget](https://osu.ppy.sh/community/forums/topics/250902)—A desktop app that allows for the end user to keep track of their rank, accuracy, and play count.

> - osu-pps—A website that lists the most overweight maps in terms of PP by using a calculation similar to [ppv1](/wiki/Performance_points/ppv1), and sorts mappers by how many overweight maps they have or how many quality maps they have according to other mappers' favorites.

Reviewer 2: ```suggestion
- osu-pps—A website that lists the most overweight maps in terms of [pp](/wiki/Performance_points) by using a calculation system similar to [ppv1](/wiki/Performance_points/ppv1), and sorts mappers by how many overweight maps they have or how many high-quality beatmaps they have according to other mappers' favorites.
```

  - [Website](https://osu-pps.com/#/osu/maps)

> - [PerformancePlus (PP+)](https://osu.ppy.sh/community/forums/topics/580994)—An unofficial osu!Standard ranking system featuring Player Rankings, both global and national, Global Score Rankings, each for 8 different (sub)skill categories, detailed player profiles, and map leaderboards for any ranked beatmap sorted by pp+ performance, and bimonthly Seasonal Rankings.

Reviewer 2: ```suggestion
- [PerformancePlus (PP+)](https://osu.ppy.sh/community/forums/topics/580994)—An unofficial osu! ranking system featuring global and national player rankings, global score rankings, rankings for 8 different (sub)skill categories, detailed player profiles, leaderboards for ranked beatmaps using the pp+ system, and bimonthly seasonal rankings.
```

  - [Website](https://syrin.me/pp+/)

> - pupper.moe—Originally a spreadsheet, now a website for an opt-in US state ranking leaderboard for osu! with some states' discord servers being listed.

Reviewer 2: ```suggestion
- pupper.moe—Originally a spreadsheet, now a website for opt-in US state ranking leaderboards for osu! with some states' Discord servers being listed.
```

  - [Website](https://pupper.moe/)
- [RTPP Displayer](https://osu.ppy.sh/community/forums/topics/685031)—Calculates and displays earned pp while playing on the fly.
  - [GitHub](https://github.com/KedamaOvO/RealTimePPDisplayer)

### Miscellaneous

> - Aiess—Provides subscriptions for mapping-related events in osu! (e.g. ranks, qualifications, nominations, suggestions, newsposts, group changes, etc).

Reviewer 2: ```suggestion
- Aiess—Provides subscriptions for mapping-related events in osu! (e.g. ranks, qualifications, nominations, suggestions, news posts, group changes, etc).
```

  - [Github](https://github.com/Naxesss/Aiess)
- [AR/OD calculator \[Android\]](https://osu.ppy.sh/community/forums/topics/227510)—Calculates the AR times and OD hit window accuracy.
- [Beatmap URL Formatter](https://osu.ppy.sh/community/forums/topics/55305)—A program to format beatmap URLs to BBCode for posting.
- [BloodyFox \[Android\]](https://osu.ppy.sh/community/forums/topics/475693)—Search and view user stats and search and download beatmaps.
- [Detect the osu! folder easily](https://osu.ppy.sh/community/forums/topics/381311)—Finds the osu! folder and report its path. Intended for devs working on IO related tasks in their applications.

> - hwchr's pick'em—A site which hosts pickems for many osu! tournaments with the option to request for a tournament.

Reviewer 2: ```suggestion
- hwchr's pick'em—A site which hosts pickems for osu! tournaments with the option to request a pickem for a tournament.
```

  - [GitHub](https://github.com/JasperDeSutter/osuElements)
- [osu!monthly revival project](https://osu.ppy.sh/community/forums/topics/684728)—The revival of the osu!monthly, like the osu!weekly, but monthly.
- [osu!player](https://osu.ppy.sh/community/forums/topics/660418)—A media player just for osu! beatmaps.
  - [Github Documentation](https://github.com/Founntain/founntain/blob/master/osuplayer.md)
- [osu!Rank](https://osu.ppy.sh/community/forums/topics/133966)—Your stats on your phone.
- [osu!search](https://osu.ppy.sh/community/forums/topics/380485)—Advanced beatmap searching
  - [Website](https://osusearch.com/)
- [osu!station](https://osu.ppy.sh/community/forums/topics/236417)—A radio station powered by beatmap music
  - [Website](https://radio.yas-online.net)
- [osu!trace](https://osu.ppy.sh/community/forums/topics/303844)—A cursor tracing program. Can be used for comparsions, show how a beatmap flows, etc.
  - [Google Drive](https://drive.google.com/drive/u/0/folders/0B1tsyXqMseXKeFFDcS1RQ2I5Z3M)
- [osu!tt](https://osu.ppy.sh/community/forums/topics/302790)—Keeps track of your game client's uptime.
  - [Github](https://github.com/Flanster/osuTT)

> - osu-trainer—A program that allows you to modify the difficulty of a beatmap very quickly and easily.

Reviewer 2: ```suggestion
- osu-trainer—A program that allows you to modify the difficulty settings of a beatmap easily.
```

  - [Github](https://github.com/FunOrange/osu-trainer)

> - osuStuff—A website for creating anime style avatars and for playing automatic osu! Battle Royale Tournaments

Reviewer 2: ```suggestion
- osuStuff—A website for creating anime-themed avatars, and playing automatic osu! Battle Royale tournaments.
```

  - [Website](https://www.osustuff.org/)
- [Script chan](https://osu.ppy.sh/community/forums/topics/730734)—Referee tool to create and manage osu! rooms.

>   - [Git](https://git.cartooncraft.fr/shARPII/script-chan/)<!-- Unsure about naming so currently left as "Git" -->

Reviewer 2: ```suggestion
  - [GitLab](https://git.cartooncraft.fr/shARPII/script-chan/)
```
